conformance_pack/codebuild.sp

```diff
@@ -102,8 +104,19 @@ control "codebuild_project_artifact_encryption_enabled" {
 
   tags = merge(local.conformance_pack_codebuild_common_tags, {
     cis_controls_v8_ig1 = "true"
-    gxp_21_cfr_part_11  = "true"
-    gxp_eu_annex_11     = "true"
+    nist_csf            = "true"
+  })
+}
+
+control "codebuild_project_s3_logs_encryption_enabled" {
```

Comment:
This is a duplicate control, please remove.

Reply:
This happened while resolving merge conflict.